framework/src/meshgenerators/MeshDiagnosticsGenerator.C

```diff
@@ -149,7 +157,11 @@ MeshDiagnosticsGenerator::checkSidesetsOrientation(const std::unique_ptr<MeshBas
 {
   auto & boundary_info = mesh->get_boundary_info();
   auto side_tuples = boundary_info.build_side_list();
-
+  auto sideset_map = boundary_info.get_sideset_map();
+  auto side_set = boundary_info.get_side_boundary_ids();
+  boundary_info.build_node_list_from_side_list();
+  auto nodeset_map = boundary_info.get_nodeset_map();
+  // auto & num_sides = mesh->n_sides();
   for (const auto bid : boundary_info.get_boundary_ids())
   {
```

Comment:
I think this did not need to change?

Reply:
I think I added that for something I was testing and just forgot to remove it.